### DISCUSSION TITLE
Disabled buttons are not shown

### DISCUSSION THREAD

**Discussion by MaRoe** [EclipseNebula/nebula#588] [Q&A] [answered]
Hi,

after getting the spellchecking working (thanks again for your help) I stumbled across another glitch. In development mode everything works like charm. As soon as  everything is exported as a stand-alone application deactivated toolbar buttons are not shown (see screenshot). If an action becomes available (for example Copy or Cut) the button appears - if it it becomes unavailable the button vanishes.

Did anyone had a similar experience? Did I miss a step while exporting the application?

Thanks in advance & cheers
Martin

![grafik](https://github.com/eclipse/nebula/assets/176706/3ac5753e-cdcc-423b-95cc-9aba3f9236cf)

**Comment by lcaron** [comment]
Hi Martin

Any news about this issue ?

Laurent

**Comment by MaRoe** [comment]
Hi Laurent,

Great to hear from you.

Unfortunately no, nothing changed on this issue.

Cheers and a great Sunday evening! 
Martin

**Comment by lcaron** [comment]
Sad to here that... @fipro78 do you have any clue ?

**Reply by fipro78** [reply]
IIRC there is an issue with regards to the alpha setting. Disabled buttons are shown with an alpha setting that is not correctly handled. 

But I would need to check the old wiki and Bugzilla. Can't remember right now. Will try to find something.

**Reply by fipro78** [reply, marked as answer]
Maybe you will find a clue here

https://www.eclipse.org/forums/index.php/t/1077407/
